examples/analog/AI_multipleChannels.py

```diff
@@ -19,11 +19,11 @@
    AI2: AI2 to AI6
    AI3: AI3 to AI7
 
-A time gap exists between the end of one signal acquisition and the start of
```

Comment:
Since it's n samples related comment, move to `Section 2: Multiple Points (n samples)`.